### ISSUE TITLE
Chained calls to #order do not work as described with ActiveRecord 4.0.0

### ISSUE THREAD

**Issue by wangteji** [panorama-ed/order_as_specified#22]
The gemspec says that this gem is compatible with ActiveRecord versions >= 4.0.0, but when used with AR 4.0.0, chained calls to order do not work as described in the README. 

Consider the following example:
```ruby
  TestClass.create(field: "bar") # => id: 1
  TestClass.create(field: "foobar") # => id: 2
  TestClass.create(field: "foo") # => id: 3
  TestClass.create(field: "baz") # => id: 4
  
  # Running on AR 4.0.0:
  TestClass.order_as_specified(field: ["foo", "bar"]).order(:id).map(&:field)
  # => ["bar", "foobar", "foo", "baz"] # Results ordered by :id
  TestClass.order(:id).order_as_specified(field: ["foo", "bar"]).map(&:field)
  # => ["foo", "bar", "foobar", "baz"] # Results ordered by specified field values, then id

  # Running on AR 4.0.1
  TestClass.order_as_specified(field: ["foo", "bar"]).order(:id).map(&:field)
  # => ["foo", "bar", "foobar", "baz"] # Results ordered by specified field values, then id 
```
This happens because in AR 4.0.0, each additional call to #order prepends to the ORDER BY clause. This behavior was changed in AR 4.0.1 as described here http://weblog.rubyonrails.org/2013/11/1/Rails-4-0-1-has-been-released/. 

The gem will still work fine with 4.0.0 as long as you're aware of this difference in how the chained calls to order work. I think the README may confuse some people though, since it assumes the AR 4.0.1 behavior. Also, the tests break when run while using AR 4.0.0.

**Comment by JacobEvelyn** [comment]
Thanks so much for finding this issue. I think the best solution here is to just update the dependency version to Rails 4.0.1. I'll go ahead and do that and then release a new gem version.

**Comment by JacobEvelyn** [comment]
This is fixed in version 1.2. Thanks again!
